vizro-ai/docs/pages/user_guides/model_config.md

```diff
@@ -45,4 +45,31 @@ cost. In general, the `gpt-3.5-turbo` collection provides the most cost-effectiv
 which would be a good starting point for most users. `gpt-4` models are more powerful than `gpt-3` models, e.g. they allow for more tokens per request. You can refer to these models' [capabilities](https://platform.openai.com/docs/models/overview)
 and [pricing](https://openai.com/pricing) for more information.
 
+### Azure OpenAI models
```

Comment:
Again I'm curious how general this solution is. It feels like it should work for more than just Azure OpenAI, which is just one example?

Reply:
It could work for any model in theory,  but it should not be allowed in practice. 
- In code it will work without throwing an error as langchain allows to add any models here(google ones, cohere, anything that langchain supports).
- but for supported LLM, we only have gpt3.5 and gpt4 with openai right now since we are using `openai function calling` in all LLM calls. No other models are supported at this moment. 

tbh with regards to this, it would probably be better to use the property.setter than doing this, it leads to a bit different conclusion from our discussion last time if we want to be more vigilant in terms of what should be allowed here in `vizro-ai`